augly/image/intensity.py

```diff
@@ -286,6 +285,8 @@ def sharpen_intensity(factor: float, **kwargs) -> float:
 def shuffle_pixels_intensity(factor: float, **kwargs) -> float:
     return factor * 100.0
 
+def skew_intensity(level: float, **kwargs) -> float:
+    return min(level, 100)
```

Comment:
can you normalize the output of this function to be from [0, 100]. it's currently from [-inf, 100]

Reply:
also
```suggestion
    return min(level, 100.0)
```